server/src/main/java/CSC4410/CovidTracker/controller/LocationSearchController.java

```diff
@@ -0,0 +1,26 @@
+package CSC4410.CovidTracker.controller;
+
+import CSC4410.CovidTracker.model.County;
+import CSC4410.CovidTracker.operation.query.CountyLocationQuery;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLException;
+
+@RestController
+public class LocationSearchController {
+
+    private final CountyLocationQuery query = new CountyLocationQuery(0, 0);
```

Comment:
I think is better if you inject this object by using annotations and because it is final try to instantiate it inside a constructor.
Example
 private final CountyLocationQuery query;
@autowire
public LocationSearchController(){
this. query = new CountyLocationQuery(0, 0);
}